adapter/noop2/noop2.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Istio Authors.
```

Comment:
The new will be done on the registration code (it will be responsibility of the Adapter to new on it's object).. configure will be to add more data to the adapter object to make it complete. I think we do not need NewAdapter .